apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -246,19 +246,19 @@
 
         <!-- region bank ledgers-->
         <div class="clearfix"></div>
-        <h1 class="mr-t2 primary_clr" *ngIf="isBankTransactionLoading">Loading Transactions...</h1>
+        <h1 class="mr-t2 primary-color" *ngIf="isBankTransactionLoading">Loading Transactions...</h1>
         <section id="eledgerwrap" class="mr-t2" *ngIf="lc.showEledger">
-            <div class="alert alert-danger ledger-book fade in pr" style="padding-right:15px">
+            <div class="alert alert-danger ledger-book fade in position-relative pr-15">
```

Comment:
you are added two type of padding classes like **pr-15** and **pd-r2** it all suld be same like pd-15 , pd-2 or 
pd-r15, pd-r2 same for margin. make it similar please because designer/developer have to check again and again on file which class should be added for margin and padding. if it is a similar naming convention it would be easy to us.

Reply:
done